phoenix-core/src/main/java/org/apache/phoenix/transaction/NotAvailableTransactionProvider.java

```diff
@@ -60,7 +60,7 @@ public PhoenixTransactionClient getTransactionClient(Configuration config, Conne
 
     @Override
     public Provider getProvider() {
-        return TransactionFactory.Provider.TEPHRA;
+        return TransactionFactory.Provider.OMID;
```

Comment:
This is not right.
We should either return null, or the UnavailableProvider's type, if we keep that,